08-CMSIS/Makefile

```diff
@@ -0,0 +1,123 @@
+CROSS_COMPILE ?= arm-none-eabi-
+CC := $(CROSS_COMPILE)gcc
+AS := $(CROSS_COMPILE)as
+CORTEX_M3 := cortex-m3
+CORTEX_M4 := cortex-m4
```

Comment:
Use the similar techniques to distinguish CFLAGS for different targets as F9 does: https://github.com/f9micro/f9-kernel/blob/master/platform/build.mk